client/web/src/Structure/Questions/State/action.js

```diff
@@ -0,0 +1,34 @@
+import { questionConstant } from './actionTypes'
+import axios from 'axios'
+
+const token = localStorage.getItem('token')
+
+export const nextQuestionLoading = () => ({
+    type: questionConstant.GET_NEXT_QUESTION_LOADING
+})
+
+export const nextQuestionFailure = payload => ({
+    type: questionConstant.GET_NEXT_QUESTION_FAILURE,
+    payload
+})
+   
+export const nextQuestionSuccess = payload =>  ({
+    type: questionConstant.GET_NEXT_QUESTION_SUCCESS,
+    payload
+})
+
+export const nextQuestion = ({ attemptId, submissionId }) => dispatch => {
+    dispatch(nextQuestionLoading())
+    axios({
+        method: 'POST',
+        url: `${process.env.REACT_APP_ATTEMPT_URL}/next`,
+        headers: {
```

Comment:
Don't directly use process.env.REACT_APP_ATTEMPT_URL assign it to a variable and use